src/scripts/checkVersion.js

```diff
@@ -0,0 +1,31 @@
+/* eslint-disable no-console */
+
+const { execSync } = require('child_process');
+
+const checkVersion = (command, name, expected) => {
+  let version = null;
+  try {
+    version = execSync(command).toString().replace('v', '').trim();
+  } catch (e) {
+    console.log(`Unable to get ${name} version!  😱`);
+    console.log('');
+    process.exit(1);
+  }
+  if (version !== expected) {
+    const contributingLink = 'https://github.com/NimaSoroush/differencify/blob/master/CONTRIBUTING.md';
+    console.log(`Expected ${name} version ${expected}, but got ${version}.  😱`);
+    console.log('');
+    console.log(`Please follow Differencify's contributing guide (see ${contributingLink}).`);
+    console.log('');
+    process.exit(1);
+  }
+};
+
+console.log('Checking Node & npm versions...');
+console.log('');
+
+checkVersion('node --version', 'Node', '8.6.0');
```

Comment:
does this need to be this strict?

Reply:
This is for development of this package. I want to make sure people run and test with same page and not getting different results. This been added to .nvmrc too

Reply:
I've added a note to contributing.md as well

Reply:
yeah, it's fine, I just discovered you can do `nvm use` with nvmrc files.